docs/source/deployment/netlify.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Deploy a fullstack Apollo app with Netlify
+sidebar_title: Netlify
+description: How to run your API using Netlify Functions on AWS Lambda
+---
+
+[Netlify functions](https://www.netlify.com/docs/functions/), is a new way to deploy server-side code to AWS Lambda alongside your frontend, right from GitHub. In this guide, I'll show you why Netlify's functions is a great way to run your whole Apollo app.
+
+1. Use the same platform to deploy your frontend and API code.
+2. Deploy previews of both your frontend and API on every pull request.
+3. No need to worry about scaling containers due to a serverless architecture.
+
+We'll set up a "Hello World" API and frontend for an Apollo app on Netlify. Let's get started!
+
+## Download and Run the Netlify Starter Kit
+
+Download and run the [official Netlify create-react-app and Lambda starter kit](https://github.com/netlify/create-react-app-lambda). This will set us up with a base for our frontend and API code.
```

Comment:
```suggestion
Download and run the [official Netlify `create-react-app` and Lambda starter kit](https://github.com/netlify/create-react-app-lambda). This will set us up with a base for our frontend and API code.
```